### PR TITLE
feat: configurable k6 setup timeout

### PR DESCRIPTION
**Pull request by adriantam**
## Description
Add configurable k6 setup timeout.

## References
Close https://github.com/openfga/helm-charts/issues/14

## Review Checklist
- [X] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [X] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: charts/benchmark/values.yaml [hunk 1]
```diff
@@ -8,4 +8,5 @@ Mparam: 1
 Kparam: 1
 testCase: 1
 testCheck: true
-testListObject: false
\ No newline at end of file
+testListObject: false
+setupTimeout: 1200
```
Comment (miparnisari) [inline charts/benchmark/values.yaml:12]: in seconds?
```suggestion
setupTimeoutInSeconds: 1200
```

File: charts/benchmark/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: benchmark
 description: A Kubernetes Helm chart to deploy OpenFGA and run the standard benchmark suite against it.
 
 type: application
-version: 0.0.3
+version: 0.0.4
 appVersion: "v0.4.2"
```
Comment (jon-whit) [inline charts/benchmark/Chart.yaml:7]: Let's also bump our appVersion to the latest release `v0.4.3` as well.

File: charts/benchmark/values.yaml (hunk 1 above)
Comment (adriantam) [reply charts/benchmark/values.yaml]: changed

File: charts/benchmark/Chart.yaml (hunk 2 above)
Comment (adriantam) [reply charts/benchmark/Chart.yaml]: done

File: charts/benchmark/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (adriantam) [reply charts/benchmark/values.yaml]: done

**Review by jon-whit — APPROVED** [review]

**Pull request merged**